frontend/src/components/AcmDataForm.tsx

```diff
@@ -469,6 +474,11 @@ export function AcmDataFormWizard(props: {
             ),
             component: section.type === 'Section' && (
                 <Form isHorizontal={isHorizontal}>
+                    {operatorError && (
```

Comment:
One question I have about this alert, it currently does not prevent the the wizard from creating an automation template. GRC's alert also does not prevent creation. Is this the behavior we want?